app/controllers/gardens_controller.rb

```diff
@@ -1,6 +1,9 @@
 class GardensController < ApplicationController
   before_action :require_user, except: [:show]
 
+  def index
+    @gardens = GardenFacade.create_garden_objects(current_user.gardens)
+  end
```

Comment:
Little confused as to why we are returning the user's gardens here? I assume when this action is hit, we would want to display all gardens between all users that aren't listed as private.

Reply:
@hashmaster3k Updated this to remove gardens index and added this as an extension story to our board. Now user just has dashboard.